src/styles/blocks/page.scss

```diff
@@ -0,0 +1,50 @@
+.page {
+  scroll-behavior: smooth;
+
+  &__body {
+    margin: 0;
+    background-color: $mainBackgroundColor;
+    overflow-x: hidden;
+  }
+
+  &__body:has(.page__menu:target) {
+    overflow: hidden;
+  }
+
+  &__menu {
+    position: fixed;
+    top: 0;
+    right: 0;
+    left: 0;
+
+    transform: translateX(-100%);
+    transition: transform 0.3s;
+
+    &:target {
+      transform: translateX(0);
+    }
+  }
+
+  &__section {
+    margin-bottom: 90px;
+    font-family: "Poppins", sans-serif;
+
+    @include onTablet {
+      margin-bottom: 132px;
+    }
+
+    @include onDesktop {
+      margin-bottom: 164px;
+    }
+
+    &-title {
+      margin: 0 0 40px;
+      font-weight: 500;
+      font-size: 32px;
+      line-height: 32px;
+      text-align: center;
+      letter-spacing: -1.5px;
+      color: $mainColor;
+    }
```

Comment:
This needs to be outside of page__section because it is not a modifier. Also title sizes needs to chance depending on the screen sizes.

Reply:
Yep, this didn't feel right at the time either. Fixed now.